// Copyright 2021 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto2";

package feedwire;

option optimize_for = LITE_RUNTIME;

enum FeedEntryPointSource {
  CHROME_DISCOVER_FEED = 19;
  CHROME_FOLLOWING_FEED = 20;
  CHROME_SINGLE_WEB_FEED_MENU = 24;
  CHROME_SINGLE_WEB_FEED_ATTRIBUTION = 25;
  CHROME_SINGLE_WEB_FEED_RECOMMENDATION = 26;
  CHROME_SINGLE_WEB_FEED_GROUP_HEADER = 28;
  CHROME_SINGLE_WEB_FEED_OTHER = 27;
}
